### ISSUE TITLE
I can't start the webapp due to a TypeError

### ISSUE THREAD

**Issue by oren** [s4y/space#26]
I am on Firefox 99.0.1 (64-bit). When I go to 127.0.0.1:8031 I see the prompt for allowing my video camera. After I approve it i see black screen with a textbox at the bottom of the screen.

Here is the javascript error in the JS console:

```
Uncaught (in promise) TypeError: setting getter-only property "ac"
    get ac http://127.0.0.1:8031/space/service/room.js:123
    get ac http://127.0.0.1:8031/space/service/room.js:309
    start http://127.0.0.1:8031/:857
    promise callback*get http://127.0.0.1:8031/space/js/Service.js:11
    start http://127.0.0.1:8031/:856
    promise callback*start http://127.0.0.1:8031/:816
    <anonymous> http://127.0.0.1:8031/:870
room.js:123:17
    get ac http://127.0.0.1:8031/space/service/room.js:123
    get ac http://127.0.0.1:8031/space/service/room.js:309
    start http://127.0.0.1:8031/:857
    (Async: promise callback)
    get http://127.0.0.1:8031/space/js/Service.js:11
    start http://127.0.0.1:8031/:856
    (Async: promise callback)
    start http://127.0.0.1:8031/:816
    <anonymous> http://127.0.0.1:8031/:870
    InnerModuleEvaluation self-hosted:2331
    evaluation self-hosted:2292
```

**Comment by s4y** [comment]
Thanks for finding and trying this project! This issue has now been fixed by 9549fe87e90ea0ae5a23acadb463acdbaa3973a3, let me know if that helps.

This reminds me that I have a few changes I'd like to port in from forks of this project that would significantly improve the look and feel, so I apologize in advance for any chunkiness.

**Comment by oren** [comment]
thank you! it's running now. is there a way to run it on my ubuntu laptop? just for testing with friends
